test/assert.js

```diff
@@ -792,13 +792,44 @@ describe('assert', function () {
 
     err(function() {
       assert.isAbove(null, 1);
-    }, 'expected null to be a number');
+    }, 'expected null to be a number or a date');
 
     err(function() {
       assert.isAbove(1, null);
     }, 'the argument to above must be a number');
   });
 
+  it('above (dates)', function() {
+    var now = new Date();
+    var oneSecondAgo = new Date(now.getTime() - 1);
+    assert.isAbove(now, oneSecondAgo, 'Now should be above 1 second ago');
+
+    err(function() {
+      assert.isAbove(oneSecondAgo, now);
+    }, 'expected ' + now.toUTCString() + ' to be above ' + oneSecondAgo.toString()); // TODO: Fix different date formatting
```

Comment:
The error message is created by `util.getMessage` inside the `Assertion.prototype.assert` [here](https://github.com/chaijs/chai/blob/616cf8bf883afa67f99a952901b4b44fcabc0cfe/lib/chai/assertion.js#L139).

The `util.getMessage` uses [util.objectDisplay](https://github.com/chaijs/chai/blob/616cf8bf883afa67f99a952901b4b44fcabc0cfe/lib/chai/utils/getMessage.js#L46) which ends up calling [util.inspect](https://github.com/chaijs/chai/blob/616cf8bf883afa67f99a952901b4b44fcabc0cfe/lib/chai/utils/objDisplay.js#L28) that for Dates [will call toUTCString()](https://github.com/chaijs/chai/blob/616cf8bf883afa67f99a952901b4b44fcabc0cfe/lib/chai/utils/inspect.js#L124) 😄 

But the `.above | .below` assertions are just [concatenating](https://github.com/chaijs/chai/blob/616cf8bf883afa67f99a952901b4b44fcabc0cfe/lib/chai/core/assertions.js#L1101) the expected value to the message which just calls `.toString()`.

In order to fix that you need to update the call to `this.assert`.

Unfortunatelly github does not allow me to comment on unchanged lines so I'll try to explain here.

Basically you need to update this:
```js
this.assert(
  obj > n
  , 'expected #{this} to be above ' + n
  , 'expected #{this} to be at most ' + n
);
```
To this:
```js
this.assert(
  obj > n
  , 'expected #{this} to be above #{exp}'
  , 'expected #{this} to be at most #{exp}'
  , n
);
```
I believe you will need to do the same thing for the .below assertion too

Please let me know if you have any further questions 😄